package src.Exceptions;

public class AlreadyAuthenticatedException extends Throwable{
}
